lib/spack/spack/test/cmd/graph.py

```diff
@@ -30,43 +30,41 @@
 graph = SpackCommand('graph')
 
 
-def test_graph_ascii(builtin_mock, database):
-    """Tests spack graph --ascii"""
+@pytest.mark.db
+@pytest.mark.usefixtures('builtin_mock', 'database')
+class TestGraphCommand(object):
```

Comment:
Do you prefer to revert gathering tests in a class also here and in the other two commands?

Reply:
Yeah kind of — is there a good reason to use classes?  My impression was that the pytest style preferred modules

Reply:
> is there a good reason to use classes?

I just used classes to gather test properties together. Here, for instance, there's one test that doesn't need `builtin_mock` and `database` + I had to specify those 2 fixtures just once.

But really, no strong objections if you want to revert this.